### ISSUE TITLE
closeAllConnections in getRetrosheet breaks knitr

### ISSUE THREAD

**Issue by cantpitch** [colindouglas/retrosheet#1]
Similar to this issue: https://stackoverflow.com/questions/41372927/isincompletecon-error-when-knitting-pdf, the `closeAllConnections` calls in `getRetrosheet` currently break `knitr` with the error:

```
Quitting from lines 105-106 (Chapter04.Rmd) 
Error in isIncomplete(con) : invalid connection
Calls: <Anonymous> ... evaluate_call -> handle_output -> <Anonymous> -> isIncomplete
Quitting from lines 105-106 (Chapter04.Rmd) 
Error in isOpen(con) : invalid connection
Calls: <Anonymous> ... <Anonymous> -> evaluate_call -> <Anonymous> -> isOpen
Execution halted
```

**Comment by colindouglas** [comment]
Thanks for the report! This is caused by sloppy connection handling (on my part). I've pushed some small changes (ffd76ea9302479ec68ae3d4fe54894f6b23c99d2) to get rid of the `closeAllConnections()` calls and instead close the specific connections. I'm going to hold off pushing these changes to CRAN until you can verify that it's working for you. Please use the development version:

`devtools::install_github("colindouglas/retrosheet")`

 and let me know if this resolves your issue.. Also, please also let me know if you are encountering any "closing unused connection" warning messages.

**Comment by colindouglas** [comment]
Should be resolved in [v1.1.2 ](https://github.com/colindouglas/retrosheet/releases/tag/1.1.2)
